exporter.go

```diff
@@ -1634,6 +1636,10 @@ func (e *Exporter) scrapeRedisHost(ch chan<- prometheus.Metric) error {
 
 	e.extractCountKeysMetrics(ch, c)
 
+	if strings.Contains(infoAll, "Sentinel") {
```

Comment:
Do you have an example for me what exactly is being matched here? What's the INFO string? Can you narrow it down more or is `Sentinel` the best we can do?

Reply:
Below is an example INFO string section for sentinels (only presents in redis-sentinel instances, not in redis instance) -

```
# Sentinel
sentinel_masters:1
sentinel_tilt:0
sentinel_running_scripts:0
sentinel_scripts_queue_length:0
sentinel_simulate_failure_flags:0
master0:name=mymaster,status=ok,address=172.17.0.2:6379,slaves=3,sentinels=3

```

We are matching `Sentinel` from this INFO ALL string. 

I think this is the best we can do here, since we are already getting INFO ALL string.

Reply:
Yeah, i think doing this here is fine. But can you match for `# Sentinel`?

Reply:
Match condition was updated to `# Sentinel` from `Sentinel`.